cpp/src/arrow/compute/kernels/vector_replace.cc

```diff
@@ -73,6 +73,10 @@ struct CopyArrayBitmap {
   const uint8_t* in_bitmap;
   int64_t in_offset;
 
+  const uint8_t* GetInBitmap() const {
+    return in_bitmap;
+  }
+
```

Comment:
You can remove this method, this is a struct so  properties are public by default. I recommend to check this reference: https://www.educative.io/collection/page/10370001/5111386763952128/6533026123087872

Reply:
I add this because the CopyBitmap is a template with  typename for this struct and can't acces to the in_bitmap property.